tests/api/test_category.py

```diff
@@ -2,11 +2,40 @@
 
 import graphene
 from django.template.defaultfilters import slugify
+from unittest.mock import Mock
+
 from saleor.product.models import Category
 from tests.utils import create_image
 from tests.api.utils import get_graphql_content, get_multipart_request_body
 
 
+FETCH_CATEGORY_QUERY = """
+query fetchCategory($id: ID!){
+    category(id: $id) {
+        name
+        backgroundImage {
+           url(size: 120)
+        }
+    }
+}
+"""
+
+LEVELED_CATEGORIES_QUERY = """
```

Comment:
Could we place those queries above the tests using it? It's way easier to find what's going on in such case.